smartsim/database/orchestrator.py

```diff
@@ -379,16 +418,20 @@ def set_batch_arg(self, arg: str, value: t.Optional[str] = None) -> None:
         :param value: batch param - set to None if no param value
         :type value: str | None
         :raises SmartSimError: if orchestrator not launching as batch
-        """        
-        if not hasattr(self, 'batch_settings') or not self.batch_settings:
+        """
+        if not hasattr(self, "batch_settings") or not self.batch_settings:
             raise SmartSimError("Not running as batch, cannot set batch_arg")
 
         if arg in self._reserved_batch_args[type(self.batch_settings)]:
             logger.warning(
-                f"Can not set batch argument {arg}: it is a reserved keyword in Orchestrator"
+                (
+                    "Can not set batch argument {}: "
+                    "it is a reserved keyword in Orchestrator"
+                ),
```

Comment:
parens unnecessary?

Reply:
unnecessary but gross to look at. it could appear as a "missing comma" without additional indentation. I'm sort of on a case by case basis on the parens unnecessary items, but I am not a fan.

I'm going with f-string to avoid annoying confusion about whether arg should be part of concat vs passed to logger.warning and whether or not the second line is the replacement...